### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by chris-stytch**
# Description

Updates README for new config setup, corrects some link rot, and clarifies that this app uses the Page Router.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,15 @@
...
 
 <p align="center">
   <img src="https://user-images.githubusercontent.com/100632220/217049841-b9eeb72a-3e50-4074-839a-e64ee5d4a88c.png" width="750">
 </p>
 
 ## Overview
 
-This example application demonstrates how one may use Stytch within a Next.js application. This project was bootstrapped with [Create Next App](https://nextjs.org/docs/api-reference/create-next-app).
-
-This project uses Stytch's [Next.js SDK](https://stytch.com/docs/sdks/javascript-sdk) which provides pre-built UI components, useful React hooks, headless methods to securely interact with Stytch, and is SSR friendly. This project also utilizes Stytch's [Node Backend SDK](https://www.npmjs.com/package/stytch) for authenticating the logged in user's session.
```
Comment (mruhlin-stytch) [inline README.md:11]: Why remove this? Not necessarily against it, but it seems like potentially helpful context

File: README.md [hunk 2]
```diff
@@ -22,41 +18,26 @@ Follow the steps below to get this application fully functional and running usin
...
-
-   - Toggle on **OAuth**.
-     
-     <img width="400" alt="SDK OAuth" src="https://user-images.githubusercontent.com/100632220/217053483-e757d1aa-af18-4af3-a476-45860ca3065f.png">
+2. Within your new Project, navigate to [SDK configuration](https://stytch.com/dashboard/sdk-configuration), and click **Enable SDK**.
 
-3. Navigate to [Redirect URLs](https://stytch.com/dashboard/redirect-urls), and add `http://localhost:3000/authenticate` as the types **Login** and **Sign-up**.
-   
-   <img width="400" alt="Redirect URLs" src="https://user-images.githubusercontent.com/100632220/217983021-d8bf6fff-6a68-4e94-bffd-d062e69c8817.png">
+3. Navigate to [OAuth](https://stytch.com/dashboard/oauth), and set up login for Google in the Test environment.
```
Comment (mruhlin-stytch) [inline README.md]: Do we need this/should we re-word since Google OAuth is configured by default now in Test?

File: README.md [hunk 3]
```diff
@@ -22,41 +18,26 @@ Follow the steps below to get this application fully functional and running usin
...
-   - Within the **Email Magic Links** drawer, toggle on **Enable the LoginOrCreate Flow**.
-     
-     <img width="400" alt="SDK Email Magic Links" src="https://user-images.githubusercontent.com/100632220/217053215-8c369de8-7828-4ad6-ac88-a50918520fc3.png">
-
-   - Toggle on **OAuth**.
-     
-     <img width="400" alt="SDK OAuth" src="https://user-images.githubusercontent.com/100632220/217053483-e757d1aa-af18-4af3-a476-45860ca3065f.png">
+2. Within your new Project, navigate to [SDK configuration](https://stytch.com/dashboard/sdk-configuration), and click **Enable SDK**.
 
-3. Navigate to [Redirect URLs](https://stytch.com/dashboard/redirect-urls), and add `http://localhost:3000/authenticate` as the types **Login** and **Sign-up**.
```
Comment (mruhlin-stytch) [inline README.md:37]: I think you still need to setup redirect URLs, no?

File: README.md [hunk 4]
```diff
@@ -22,41 +18,26 @@ Follow the steps below to get this application fully functional and running usin
...
 
 In your terminal clone the project and install dependencies:
 
 ```bash
-git clone https://github.com/cal-stytch/test-stytch-nextjs-example.git
-cd test-stytch-nextjs-example
-npm i
+git clone https://github.com/cal-stytch/stytch-nextjs-example.git
+cd stytch-nextjs-example
+pnpm i
```
Comment (mruhlin-stytch) [inline README.md:41]: Do enough people use `pnpm` that we should default to that instead of `npm`?

File: README.md (hunk 3 above)
Comment (chris-stytch) [reply README.md:37]: Those should all be preset now 🙏

File: README.md (hunk 4 above)
Comment (chris-stytch) [reply README.md:41]: This was a recommendation from Vercel's reviewing engineer. 

I'm certain that npm has a much larger user base, but Next.js would like you to use pnpm. 

It does. seem worthwhile to mention alternatives though, I will add that note.

File: README.md (hunk 2 above)
Comment (chris-stytch) [reply README.md]: Yeah, fair to slightly adjust now that its config'd by default.

File: README.md (hunk 1 above)
Comment (chris-stytch) [reply README.md:11]: The create next app portion? I think the page vs app router is imo a bit more important.

Comment (chris-stytch) [reply README.md:11]: Oh, I think you may mean the paragraph below. Agreed, will add back in.

File: README.md [hunk 5]
```diff
@@ -1,19 +1,17 @@
...
+This project uses Stytch's [Next.js SDK](https://stytch.com/docs/sdks/javascript-sdk) which provides pre-built UI components, useful React hooks, headless methods to securely interact with Stytch, and is SSR friendly. This project also utilizes Stytch's [Node Backend SDK](https://www.npmjs.com/package/stytch) for authenticating the logged in user's session.
 
 This application features Email Magic Links and Google OAuth authentication. You can use this application's source code as a learning resource, or use it as a jumping off point for your own project. We are excited to see what you build with Stytch!
 
-We'd also recommend checking out our [Next.js quickstart guide](https://stytch.com/docs/guides/quickstarts/nextjs), which explains how to incorporate the Stytch authentication concepts demonstrated in this example app into your own Next.js application.
```
Comment (mruhlin-stytch) [inline README.md:15]: Why remove the quickstart link?

Comment (chris-stytch) [reply README.md:15]: Added back in, but moved. [e6690d3](https://github.com/stytchauth/stytch-nextjs-example/pull/23/commits/e6690d37d1b29a2bfac7862178cc61cc029ffc01)

**Review by mruhlin-stytch — APPROVED** [review]

**Pull request merged**